README.md

```diff
@@ -1,2 +1,17 @@
-# echo-heaps
-simple example for integrating [echo physics](https://austineast.dev/echo/) with the [heaps engine](https://heaps.io)
+# What is this
+
+- This is a game that integrating [echo physics](https://austineast.dev/echo/) with the [heaps engine](https://heaps.io)
```

Comment:
I'd prefer if this had the original text - `simple example for integrating [echo physics](https://austineast.dev/echo/) with the [heaps engine](https://heaps.io)`.

Reply:
ok, this is fixed